### PR TITLE
test: don't check destination label in RuntimePolicies with PolicyAuditMode

### LINKED ISSUE #11954: CI: RuntimePolicies: Init Policy Default Drop Test With PolicyAuditMode tests ingress
https://jenkins.cilium.io/job/Ginkgo-CI-Tests-runtime-Pipeline/430/testReport/junit/(root)/Suite-runtime/RuntimePolicies_Init_Policy_Default_Drop_Test_With_PolicyAuditMode_tests_ingress/

```
/home/jenkins/workspace/Ginkgo-CI-Tests-runtime-Pipeline/runtime-gopath/src/github.com/cilium/cilium/test/ginkgo-ext/scopes.go:512
Default policy verdict on ingress failed
Expected
    <*errors.errorString | 0xc00109e390>: {
        s: "Expected string \"[reserved:host] -> 1604 [container:somelabel] 10.15.195.27 : FORWARDED 5\" is not in the filter output of \"{.source.labels} -> {.destination.ID} {.destination.labels} {.IP.destination} : {.verdict} {.event_type.type}\": 4m0s timeout expired",
    }
to be nil
/home/jenkins/workspace/Ginkgo-CI-Tests-runtime-Pipeline/runtime-gopath/src/github.com/cilium/cilium/test/runtime/Policies.go:1558
```

[19381a3f_RuntimePolicies_Init_Policy_Default_Drop_Test_With_PolicyAuditMode_tests_ingress.zip](https://github.com/cilium/cilium/files/4746174/19381a3f_RuntimePolicies_Init_Policy_Default_Drop_Test_With_PolicyAuditMode_tests_ingress.zip)

### PR DESCRIPTION
**Pull request by tklauser**
It might rarely happen that the host starts pinging fast enough that
the endpoint still has identity "init" / 5. This currently leads to fail
the test.

As already done in the egress case, ignore the destination labels when
checking the hubble observe output.

Fixes #11954

### CODE REVIEW

**Comment by tklauser** [comment]
test-focus RuntimePolicies

**Review by gandro — APPROVED** [review]
Good catch!

File: test/runtime/Policies.go [hunk 1]
```diff
@@ -1550,11 +1550,15 @@ var _ = Describe("RuntimePolicies", func() {
...
+				// In PolicyAuditMode, this means that the ping will succeed. Therefore we don't
+				// check for the source labels in the output (they can by either [reserved:init]
+				// or [container:somelabel]), only the endpoint ID.
 				By("Testing hubble observe output")
 				// Checks for a ingress policy verdict event (type 5)
 				err := hubbleRes.WaitUntilMatchFilterLine(
-					`{.source.labels} -> {.destination.ID} {.destination.labels} {.IP.destination} : {.verdict} {.event_type.type}`,
-					fmt.Sprintf("[reserved:host] -> %s [container:somelabel] %s : FORWARDED 5", endpointID, endpointIP.IPV4))
+					`{.source.labels} -> {.destination.ID} {.IP.destination} : {.verdict} {.event_type.type}`,
+					fmt.Sprintf("[reserved:host] %s -> %s : FORWARDED 5", endpointIP.IPV4))
```
Comment (gandro) [inline test/runtime/Policies.go]: Actually, I think you want to remove the `.destination.ID` as well.

Comment (tklauser) [reply test/runtime/Policies.go]: Ah yes, thanks. I forgot to amend and push some changes which would've remove it. That's also why Travis CI failed on `make govet`.

**Review by qmonnet — APPROVED** [review]

**Review by nebril — APPROVED** [review]

**Pull request merged**